docker/k8s_tools.py

```diff
@@ -4,21 +4,21 @@
 import time
 import socket
 from kubernetes import client, config
-PADDLE_JOB_NAME = os.getenv("PADDLE_JOB_NAME")
 NAMESPACE = os.getenv("NAMESPACE")
-PORT = os.getenv("PSERVER_PORT")
 if os.getenv("KUBERNETES_SERVICE_HOST", None):
     config.load_incluster_config()
 else:
     config.load_kube_config()
 v1 = client.CoreV1Api()
 
 
-def fetch_pods_info(label_selector):
+def fetch_pods_info(label_selector, phase=None):
     api_response = v1.list_namespaced_pod(
         namespace=NAMESPACE, pretty=True, label_selector=label_selector)
     pod_list = []
     for item in api_response.items:
+        if phase is not None and item.status.phase != phase:
```

Comment:
If `phase == "Running"` then need to check if it's `Terminating`

Reply:
Done.